wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -306,11 +301,15 @@ <h1>Error: HTTP {{ status_code }} -- {{ reason }}</h1>
             {% endif %}
         </div>
 
-        {% if config.TOUCHPOINT_SCRIPT %}
```

Comment:
I'm in favor of removing this entirely. If we choose to use something like Touchpoint for feedback it should be on our feedback page. This needs to be removed from the ansible playbook.

Reply:
Will do 👍